tasks/multitenancy_migration.yml

```diff
@@ -115,11 +114,6 @@
     dest: "{{ openwisp2_path }}/multitenancy_prepare_db.py"
     mode: 0754
 
-- name: prepare DB for multitenancy migration
```

Comment:
why are you removing this?

Reply:
E301:  Commands should not change things if nothing needs doing